server/middlewares/UserValidation.js

```diff
@@ -22,6 +22,59 @@ class UserValidation {
     UserValidation.sendFormattedError(req, res, next);
   }
 
+  /**
+    * @description - This method checks if a user enters an email and password.
+    * @param {object} req - The request object bearing the email.
+    * @param {object} res - The response object sent to the next middleware.
+    * @param {object} next - The callback function to the next middleware.
+    * @returns {object} - The error object with message.
+    * @memberOf UserValidation
+    * @static
+    */
+  static validateUserLogin(req, res, next) {
+    const { email, password } = req.body;
+    const errors = {};
+    const returnError = () => res.status(422).json({
+      errors
+    });
+    if (email && password) return next();
+    if (!email && !password) {
+      errors.email = ['please enter email'];
+      errors.password = ['please enter password'];
+    }
+    if (!email) errors.email = ['please enter email'];
+    if (!password) errors.password = ['please enter password'];
+    return returnError();
+  }
+
+  /**
+    * @description - This method validates the user request body.
+    * @param {object} req - The request object to be validated.
+    * @param {object} res - Th response object to be validated.
+    * @param {object} next - The callback function to the next middleware.
+    * @returns {object} - The error object with message.
+    * @memberOf UserValidation
+    * @static
+    */
+  static validateForgot(req, res, next) {
```

Comment:
validateForgot what??

Reply:
validateForgotPassword. I will make this change now.